Assets/Scripts/SS3D/Systems/Inventory/Containers/Inventory.cs

```diff
@@ -232,7 +234,7 @@ private void CmdTransferItem(GameObject itemObject, Vector2Int position, Attache
         /// </summary>
         public void OpenContainer(AttachedContainer container)
```

Comment:
Rename to attachedContainer, no "container.Container"

Reply:
done